### PR TITLE
Add AWS EKS deployment files and documentation for K8s NIM Operator

### PR DESCRIPTION
**Pull request by edemiraydin**

### CODE REVIEW

**Review by mkhaas — COMMENTED** [review]
LGTM. Thank you so much @edemiraydin

**Review by mkhaas — APPROVED** [review]
LGTM

File: cloud-service-providers/aws/eks/README.md [hunk 1]
```diff
@@ -29,6 +29,10 @@ Note: Ensure that you are in the cloud-service-providers/aws/eks directory for t
 
 ## Cluster setup for inference:
 
+You can either use the NIM Helm Chart or NIM Operator. If you would like to use the NIM Operator, please see the instructions [here](nim-operator-setup.md).
```
Comment (supertetelman) [inline cloud-service-providers/aws/eks/README.md:32]: Try to use impersonal tense avoiding words like "you" in the technical documents.

File: cloud-service-providers/aws/eks/nim-operator-setup.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
+# NVIDIA NIM Operator on AWS EKS:
```
Comment (supertetelman) [inline cloud-service-providers/aws/eks/nim-operator-setup.md:1]: Can you add a link to this NIM Operator deployment in the main readme.md for easier discoverability?

File: cloud-service-providers/aws/eks/nim-operator-setup.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+
+Please see the NIM Operator documentation before you proceed: https://docs.nvidia.com/nim-operator/latest/index.html
+This repository is dedicated to testing NVIDIA NIM Operator on AWS EKS (Elastic Kubernetes Service).
+
+## Cluster setup for inference:
+
+To install the pre-requisites for the NIM Operator, please follow the steps below:
+
+1: Install NVIDIA Device Plugin: Install NVIDIA device plugins to run GPU workloads. Check CUDA base image version for compatibility.
```
Comment (supertetelman) [inline cloud-service-providers/aws/eks/nim-operator-setup.md:10]: Why are we installing the device plugin before the Operator? Is this necessary on EKS?

File: cloud-service-providers/aws/eks/nim-operator-setup.md [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+        "completion_tokens": 16
+    }
+}
+```
+
+# Gen-ai perf tool
+
+      kubectl apply -f perf/gen-ai-perf.yaml
+
+ssh into the triton pod
```
Comment (supertetelman) [inline cloud-service-providers/aws/eks/nim-operator-setup.md:121]: "exec into triton pod"

**Review by supertetelman — COMMENTED** [review]
Looks good with some minor formtting/style changes.

Before merging through, can you confirm that we do in fact need to install the NVIDIA device plugin separately from the installation that is included with the GPU Operator install?

**Comment by edemiraydin** [comment]
All the requested changes are made.

**Review by supertetelman — APPROVED** [review]
LGTM, thank you!

**Pull request merged**